sdk/src/Core/_bcl/ArrayEx.cs

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ * 
+ *  http://aws.amazon.com/apache2.0
+ * 
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+namespace System
```

Comment:
Since these aren't extension methods It doesn't feel right to put these Shims in `System`. Can you put the shims in `Amazon.Util/Internal/_bcl` and `Amazon.Util/Internal/_netstandard` respecitively and set the namespace to `Amazon.Util.Internal`?

Reply:
As long as you are ok if Big Integer in bouncy castle namespace gets then a util reference. That was the reason why I put it into system because that is an import that is available everywhere including the third party folder. Plus it is really just a shim API for Array

Reply:
I'm okay with the bouncy castle change. You are making a change no matter what to switch to `ArrayEx` I would rather it be explicit then it looks like it came from `System`.

Reply:
Ok will change

Reply:
Done. I have undone the BigInteger changes